resources.py

```diff
@@ -128,4 +128,30 @@ def refresh_tokens():
         token = create_token(user_id)
         return jsonify(token), 200
     except:
-        return jsonify({'error':'invalid token'}), 400
\ No newline at end of file
+        return jsonify({'error':'invalid token'}), 400
+
+@resources.route('/documents', methods=['POST'])
+def create_documents():
+    try:
+        auth_header = request.headers.get('Authorization')            
+        user_id = token_valid_check(auth_header)
+        user = get_user_by_user_id(user_id)
+        if user is None:
+            return jsonify({'message':'Invalid user_id'}), 400
+        document_id = create_document(user_id)
+        document_row = get_document_by_document_id(document_id)
```

Comment:
either: get_document (id is implied since id is the only unique key of this table), or get_document_by_id